code/modules/mob/living/simple_animal/hostile/vatbeast.dm

```diff
@@ -79,7 +79,7 @@
 	if(.)
 		return
 
-	if(owner.stat)
+	if(!owner.stat)
 		remove_ranged_ability()
 		return
```

Comment:
uhhhhh

won't this make it so that if you're conscious, this will remove your ranged ability and return out early?

Reply:
like, stat is 0 if you're conscious and >=1 if you're unconscious, in light/deep crit, dead, etc.

Reply:
oh fuck